### PR TITLE
Add consistent download section to start and end of each guide

### PR DESCRIPTION
**Pull request by 0xTim**
As discussed, this adds a consistent download section to the start and end of each guide

### CODE REVIEW

File: getting-started/cli-swiftpm/index.md [hunk 1]
```diff
@@ -227,4 +229,4 @@ _   _          _   _                                           _       _   _
 
 ---
 
-Find the source code for this guide at [https://github.com/apple/swift-getting-started-cli](https://github.com/apple/swift-getting-started-cli)
+> The source bode for this guide can be found [on GitHub](https://github.com/apple/swift-getting-started-cli)
```
Comment (Jinxiansen) [inline getting-started/cli-swiftpm/index.md:232]: Sorry, I'm just a little confused, what's the `bode`? isn't `code`?

File: getting-started/swiftui/index.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ layout: page
 title: Build an iOS app with SwiftUI
 ---
 
+> The source bode for this guide can be found [on GitHub](https://github.com/0xTim/swift-org-swiftui-tutorial)
```
Comment (Jinxiansen) [inline getting-started/swiftui/index.md:6]: Same as above `bode`.

Comment (0xTim) [reply getting-started/swiftui/index.md:6]: 🤦‍♂️

File: getting-started/cli-swiftpm/index.md [hunk 3]
```diff
@@ -3,6 +3,8 @@ layout: page
 title: Build a Command-line Tool
 ---
 
+> The source bode for this guide can be found [on GitHub](https://github.com/apple/swift-getting-started-cli)
```
Comment (0xTim) [inline getting-started/cli-swiftpm/index.md:6]:
```suggestion
> The source code for this guide can be found [on GitHub](https://github.com/apple/swift-getting-started-cli)
```

File: getting-started/cli-swiftpm/index.md (hunk 1 above)
Comment (0xTim) [inline getting-started/cli-swiftpm/index.md:232]:
```suggestion
> The source code for this guide can be found [on GitHub](https://github.com/apple/swift-getting-started-cli)
```

File: getting-started/library-swiftpm/index.md [hunk 4]
```diff
@@ -3,6 +3,8 @@ layout: page
 title: Build a library
 ---
 
+> The source bode for this guide can be found [on GitHub](https://github.com/apple/swift-getting-started-package-library)
```
Comment (0xTim) [inline getting-started/library-swiftpm/index.md:6]:
```suggestion
> The source code for this guide can be found [on GitHub](https://github.com/apple/swift-getting-started-package-library)
```

File: getting-started/library-swiftpm/index.md [hunk 5]
```diff
@@ -121,4 +123,4 @@ Test Suite 'All tests' passed at 2023-01-03 16:22:45.076.
 
 ---
 
-Find the source code for this guide at [https://github.com/apple/swift-getting-started-package-library](https://github.com/apple/swift-getting-started-package-library)
+> The source bode for this guide can be found [on GitHub](https://github.com/apple/swift-getting-started-package-library)
```
Comment (0xTim) [inline getting-started/library-swiftpm/index.md:126]:
```suggestion
> The source code for this guide can be found [on GitHub](https://github.com/apple/swift-getting-started-package-library)
```

File: getting-started/swiftui/index.md (hunk 2 above)
Comment (0xTim) [inline getting-started/swiftui/index.md:6]:
```suggestion
> The source code for this guide can be found [on GitHub](https://github.com/0xTim/swift-org-swiftui-tutorial)
```

File: getting-started/swiftui/index.md [hunk 6]
```diff
@@ -408,4 +410,4 @@ We’ve covered a lot of SwiftUI basics in this tutorial, including text, images
 
 If you’d like to continue learning SwiftUI, there are lots of free resources available. For example, [Apple publishes a wide variety of tutorials](https://developer.apple.com/tutorials/swiftui) covering essential topics, drawing and animation, app design, and more. We’ll also post links here on Swift.org to some other popular tutorials – we’re a big and welcoming community, and we’re glad to have you join!
 
-The code for this tutorial can be found [here](https://github.com/0xTim/swift-org-swiftui-tutorial).
+> The source bode for this guide can be found [on GitHub](https://github.com/0xTim/swift-org-swiftui-tutorial)
```
Comment (0xTim) [inline getting-started/swiftui/index.md:413]:
```suggestion
> The source code for this guide can be found [on GitHub](https://github.com/0xTim/swift-org-swiftui-tutorial)

```

File: getting-started/vapor-web-server/index.md [hunk 7]
```diff
@@ -3,6 +3,8 @@ layout: page
 title: Build a Web Service with Vapor
 ---
 
+> The source bode for this guide can be found [on GitHub](https://github.com/vapor/swift-getting-started-web-server)
```
Comment (0xTim) [inline getting-started/vapor-web-server/index.md:6]:
```suggestion
> The source code for this guide can be found [on GitHub](https://github.com/vapor/swift-getting-started-web-server)
```

File: getting-started/vapor-web-server/index.md [hunk 8]
```diff
@@ -157,4 +159,4 @@ $ curl http://localhost:8080/user-info -X POST -d '{"name": "Tim", "age": 99}' -
 
 Congratulations! You've built your first web server in Swift!
 
-Find the source code for this guide at [https://github.com/vapor/swift-getting-started-web-server](https://github.com/vapor/swift-getting-started-web-server)
+> The source bode for this guide can be found [on GitHub](https://github.com/vapor/swift-getting-started-web-server)
```
Comment (0xTim) [inline getting-started/vapor-web-server/index.md:162]:
```suggestion
> The source code for this guide can be found [on GitHub](https://github.com/vapor/swift-getting-started-web-server)
```

**Review by 0xTim — COMMENTED** [review]
Numptie

**Review by tomerd — APPROVED** [review]

**Comment by tomerd** [comment]
@0xTim is this ready to be merged?

**Review by alexandersandberg — APPROVED** [review]

**Comment by 0xTim** [comment]
@swift-ci test

**Comment by 0xTim** [comment]
@swift-ci test

**Pull request merged**